### PR TITLE
feat: add forma mainnet addresses

### PR DESCRIPTION
**Pull request by aroralanuk**
resolves https://github.com/hyperlane-xyz/issues/issues/1357

### CODE REVIEW

**Comment by avious00** [comment]
hey @aroralanuk what's left here to merge in? i can have @paulbalaji or @Mo-Hussain pick up this PR

**Comment by avious00** [comment]
hey @paulbalaji when you have a second could you make sure the warp deployment info for forma and sketchpad here https://github.com/forma-dev/hyperlane-bridge-ui/blob/main/src/consts/tokens.ts is reflected in the config here

i added forma metadata and ill get the address info from codecrafting

File: chains/forma/logo.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+</g>
+<defs>
+<clipPath id="clip0_163_3">
+<rect width="189.952" height="189.952" fill="white"/>
+</clipPath>
+<clipPath id="clip1_163_3">
+<rect width="189.952" height="189.952" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (paulbalaji) [inline chains/forma/logo.svg:17]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="190" height="190" fill="none"><g clip-path="url(#a)"><g clip-path="url(#b)"><path fill="#FF6B6B" d="M64 64h103v16c0 8.7-7.5 15.5-16 15.5H95V190H79c-6 0-15-8-15-16V64Z"/><path fill="#000" d="M0 0h189.952v25.902c0 14.306-11.597 25.903-25.903 25.903H0V0Z"/><path fill="#000" d="M51.805 0v189.952H25.903C11.597 189.952 0 178.355 0 164.049V0h51.805Z"/></g></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h189.952v189.952H0z"/></clipPath><clipPath id="b"><path fill="#fff" d="M0 0h189.952v189.952H0z"/></clipPath></defs></svg>
```

File: chains/forma/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - apiUrl: https://explorer.forma.art/api
+    family: blockscout
+    name: Forma Explorer
+    url: https://explorer.forma.art/
+chainId: 984122
+displayName: Forma
+displayNameShort: Forma
+domainId: 984122
+isTestnet: true
+name: sketchpad
```
Comment (paulbalaji) [inline chains/forma/metadata.yaml:12]:
```suggestion
name: forma
```

File: chains/forma/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+blockExplorers:
+  - apiUrl: https://explorer.forma.art/api
+    family: blockscout
+    name: Forma Explorer
+    url: https://explorer.forma.art/
+chainId: 984122
+displayName: Forma
+displayNameShort: Forma
+domainId: 984122
+isTestnet: true
```
Comment (paulbalaji) [inline chains/forma/metadata.yaml:11]:
```suggestion
```

File: chains/forma/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.forma.art/api
+    family: blockscout
+    name: Forma Explorer
+    url: https://explorer.forma.art/
```
Comment (paulbalaji) [inline chains/forma/metadata.yaml:6]:
```suggestion
    url: https://explorer.forma.art
```

File: chains/sketchpad/metadata.yaml [hunk 5]
```diff
@@ -19,4 +19,4 @@ nativeToken:
   symbol: TIA
 protocol: ethereum
 rpcUrls:
-  - http: https://rpc.sketchpad-1.forma.art
+  - http: https://rpc.sketchpad-1.forma.art
```
Comment (paulbalaji) [inline chains/sketchpad/metadata.yaml:22]:
```suggestion
  - http: https://rpc.sketchpad-1.forma.art

```

**Review by paulbalaji — APPROVED** [review]

**Comment by avious00** [comment]
Forma is the first end to end permissionless chain on Hyperlane, from deployment to relaying 👏

**Pull request merged**